browser-tests/imposter/config/api-config.yaml

```diff
@@ -213,3 +218,107 @@ resources:
         {
           "page": "prove-identity-another-type-photo-id"
         }
+
+  # Journeys where there is an existing identity
+  # After initialising the session, the first endpoint hit is /journey/next which we stub here
+  # to return page-ipv-reuse to mimic the reuse journey
+  - method: POST
+    path: /journey/next
+    requestHeaders:
+      content-type: "application/json"
+      ipv-session-id:
+        value: "reuseJourney"
+        operator: Contains
+    queryParams:
+      currentPage:
+        operator: NotExists
+    response:
+      statusCode: 200
+      content: |
+        {
+          "page": "page-ipv-reuse"
+        }
+
+  # After initialising the session, the first endpoint hit is /journey/next which we stub here
+  # to return the confirm-your-details page to mimic the repeat fraud check journey
+  - method: POST
+    path: /journey/next
+    requestHeaders:
+      content-type: "application/json"
+      ipv-session-id:
+        value: "fraudCheckJourney"
+        operator: Contains
+    queryParams:
+      currentPage:
+        operator: NotExists
+    response:
+      statusCode: 200
+      content: |
+        {
+          "page": "confirm-your-details"
+        }
+
+  - method: GET
+    path: /user/proven-identity-details
+    requestHeaders:
+      content-type: "application/json"
+      ipv-session-id:
+        value: "KennethDecerqueira"
+        operator: Contains
```

Comment:
I'm a little nervous about using `Contain` in this configuration - it starts to make this config harder to reason about and we don't want a general simulation of core back, just the specific routes we expect to test.
That said, I can see that it saves a fair bit of duplication here, but I think we need to keep an eye on it and not let it get too clever.